### ISSUE TITLE
Future Accessibility Tasks

### ISSUE THREAD

**Issue by hdhanani7** [mobify/pinny#33]
- [ ] `Esc` key should be mapped to closing the modal
- [ ] A (perhaps overridable) description div should be included always - it's purpose is to describe the modal and how to interact with it. Implementation on the rendered html would require the description div having a unique id, and that id referred to on the pinny in a `aria-describedby` attribute

``` html
<section class="pinny" describedby="pinnyDescription" role="dialogue">
    <div id="pinnyDescription" class="visually-hidden"><!-- description text --></div>
    <!-- .pinny__header -->
    <!-- .pinny__content -->
</section>
```
